vtStor/DataCommandFormater.h

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
It used contain structure of element of data command. Ex : LBA range (8byte). I call DataCommand